src/vs/workbench/parts/preferences/browser/settingsTree.ts

```diff
@@ -1059,14 +1067,31 @@ export class SettingsRenderer implements ITreeRenderer {
 			template.descriptionElement.setAttribute('checkbox_label_target_id', baseId + '_setting_item');
 		}
 
+		template.otherOverridesElement.innerHTML = '';
+
 		if (element.overriddenScopeList.length) {
 			const otherOverridesLabel = element.isConfigured ?
 				localize('alsoConfiguredIn', "Also modified in") :
 				localize('configuredIn', "Modified in");
 
-			template.otherOverridesElement.textContent = `(${otherOverridesLabel}: ${element.overriddenScopeList.join(', ')})`;
-		} else {
-			template.otherOverridesElement.textContent = '';
+			DOM.append(template.otherOverridesElement, $('span', null, `${otherOverridesLabel}: `));
+
+			for (let i = 0; i < element.overriddenScopeList.length; i++) {
+				let view = DOM.append(template.otherOverridesElement, $('a.modified-scope', null, element.overriddenScopeList[i]));
+
+				if (i !== element.overriddenScopeList.length - 1) {
+					DOM.append(template.otherOverridesElement, $('span', null, ', '));
+				}
+
+				DOM.addStandardDisposableListener(view, DOM.EventType.CLICK, (e: IMouseEvent) => {
+					this._onDidClickOverrideElement.fire({
+						targetKey: element.setting.key,
+						scope: element.overriddenScopeList[i]
```

Comment:
Won't this always reference the last item in the list, because it will point to `i` in the closure scope, which is always incremented to the length of the list?

Reply:
Huh, that is a fantastic point, let me test it really quick.

Reply:
No, testing locally seems to indicate that when it creates the listener and anonymous function is retains the state of the scoped variable. When I manually pushed in a second option it properly used the correct index on click.

Reply:
You're right - seems that this works with `let` but does not work with `var`. I can't drop the old school habits.